examples/sending_concurrent_transactions/main.go

```diff
@@ -0,0 +1,143 @@
+// performance_transaction shows how to improve performance of the transaction submission of a single transaction
```

Comment:
performance_transaction -> `sending_concurrent_transactions`
(and in .gitignore)